atlasdb-client/src/test/java/com/palantir/atlasdb/stream/BlockConsumingInputStreamTest.java

```diff
@@ -227,12 +227,12 @@ public void get(long firstBlock, long numBlocks, OutputStream destination) {
             }
 
             @Override
-            public int expectedLength() {
+            public int expectedBlockLength() {
                 return 1_000_000;
             }
         };
 
-        // Should fail, because bigGetter.expectedLength() * blocksInMemory > Integer.MAX_VALUE.
+        // Should fail, because bigGetter.expectedBlockLength() * blocksInMemory > Integer.MAX_VALUE.
         BlockConsumingInputStream.create(bigGetter, 9001, 2148);
```

Comment:
we should assert that `bigGetter.expectedBlockLength() * blocksInMemory > Integer.MAX_VALUE` so that someone doesn't break test's assumptions